spec/JSON-RPC.md

```diff
@@ -2245,6 +2246,38 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## net_recentNetworkUsage
+Gets the recent network usage.
+It returns an object.
+The key of the object is a meaningful separation, but what is used as a key depends on the implementation.
```

Comment:
I think this part sounds a bit awkward. Let's try fixing it.

Reply:
I changed it to `The key of the object is a string, but what the keys are depend on the implementation.`